src/components/Main/MainStyles.ts

```diff
@@ -0,0 +1,309 @@
+    .container {
+      max-width: 800px;
+    }
+  }
+  @media (max-width: 767px) {
+    padding-top: 40px;
+  }
+`;
+
+const Team = styled(Section)`
+  background-image: url(${teamBg});
+  height: calc(100vw / 1.85);
+  .team-col {
+    margin-top: -60px;
+  }
+
+  @media (min-width: 1025px) {
+    .container {
+      max-width: 860px;
+    }
+  }
+
+  @media (max-width: 1024px) {
+    height: auto;
+    min-height: calc(100vw / 1.85);
+    padding-top: 100px;
+
+    .container {
+      max-width: 530px;
+    }
+  }
+`;
+
+const Member = styled(Row)`
+  background-color: #fff;
+  max-width: 400px;
+  border-radius: 20px;
+  box-shadow: 0 14px 28px rgb(0 0 0 / 25%), 0 10px 10px rgb(0 0 0 / 22%);
+  margin-bottom: 40px;
+  margin-left: 0;
+  margin-right: 0;
+  img {
+    padding: 30px;
+    padding-left: 15px;
+    height: 192px;
+    width: 170px;
+    object-fit: cover;
+    @media (max-width: 1024px) {
+      height: 192px;
+      width: 207px;
+      padding-right: 15px;
+      padding-bottom: 0;
+    }
+  }
+  .card-body {
+    padding-top: 30px;
+    padding-bottom: 30px;
+  }
+  @media (max-width: 1024px) {
+    width: 238px;
+    .col-12 {
+      flex: 0 0 100%;
+      max-width: 100%;
+    }
+  }
+  @media (max-width: 1024px) {
+    margin-left: auto;
+    margin-right: auto;
+  }
+`;
+
+const Game = styled(Card)`
+  border: none;
+  border-radius: 20px;
+  margin-bottom: 50px;
+  box-shadow: rgb(0 0 0 / 25%) 0px 14px 28px, rgb(0 0 0 / 22%) 0px 10px 10px;
+
+  img {
+    border-radius: 20px;
+    object-fit: cover;
+    object-position: center;
+    width: 100%;
+    height: 100%;
+  }
+  .card-body {
+    position: absolute;
+    bottom: 0;
+    right: 20px;
+  }
+  .card-title {
+    font-weight: 600;
+  }
+
+  @media (max-width: 400px) {
+    margin-top: 0 !important;
+    margin-left: 0 !important;
+    width: 280px !important;
+    height: calc(280px * 0.67887) !important;
+  }
+  @media (min-width: 401px) and (max-width: 768px) {
+    width: 300px;
+    height: 200px;
+    img {
+      object-position: top;
+    }
+  }
+`;
+
+const GameCol = styled(Col)`
+  @media (max-width: 992px) {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+  }
+`;
+
+const Game1 = styled(Game)`
+  @media (min-width: 1321px) {
+    width: 355px;
+    height: 241px;
+  }
+  @media (min-width: 768px) and (max-width: 1321px) {
+    width: 280px;
+    height: calc(280px * 0.67887);
+  }
+`;
+
+const Game2 = styled(Game)`
+  margin-bottom: 0;
+  @media (min-width: 1321px) {
+    width: 360px;
+    height: 312px;
+  }
+
+  @media (min-width: 768px) and (max-width: 1320px) {
+    width: 300px;
+    height: calc(300px * 0.86666);
+  }
+`;
+
+const Game3 = styled(Game)`
+  @media (min-width: 1321px) {
+    width: 522px;
+    height: 305px;
+  }
+  @media (min-width: 768px) and (max-width: 1320px) {
+    width: 414px;
+    height: calc(414px * 0.58095);
+  }
+`;
+
+const Game4 = styled(Game)`
```

Comment:
export const Game4.... will save 15 lines of the code that are in the end of the File